utils/unit_conversion/concentrations.py

```diff
@@ -0,0 +1,14 @@
+# Copyright Clinton Fernandes (clint.fernandes@gmail.com) 2021
+
+
+def ppb_to_ppm(measurement) -> float:
+    return measurement / 1000
+
+# Change micro grams per meter cubed
+# To grams per litre
+def μgm3_to_gpl(measurement)->float:
+    return measurement / 1000
+
+# 0.1 liter to 1 liter
```

Comment:
This conversion does not operate according to the comment.

0.1 litres is _already_ in litres. No conversion is necessary

Reply:
The original data was in [Num of particles] / 0.1 L
To convert to it to [Num of particles] / L, we need to multiply 10 for the number.

Reply:
But you're dividing by 1000, you're not multiplying by 10.

Reply:
μgm3_to_gpl() converts ```micro grams per meter cubed``` into ```grams per litre```,
while
pms_gt_output_to_si() changes  ```particles per 0.1 Litre``` into ```particles per Litre```.

Which function do you think is not necessary?

Reply:
I've been mis-reading this code from the very start. My apologies.